src/pages/archive/Archive.jsx

```diff
@@ -1,7 +1,49 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import {
+  NotesWrapper,
+  NoNotesMsg,
+  ButtonToNoteNow,
+} from "../home/homeComponents";
+import { ArchiveIcon } from "../../assets/icons";
+import { Link } from "react-router-dom";
+import { getArchiveNotes } from "../../Redux/Reducers/notesSlice";
+import { EachNote } from "../../components";
 
 export const Archive = () => {
+  const { archive } = useSelector((store) => store.notes);
+  const dispatch = useDispatch();
+  const [isLoading, setLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 800);
+  }, []);
```

Comment:
Seems like you are trying to show a fake loader?

Reply:
yep

Reply:
okay, but correct it later.